pkgs/development/python-modules/tkinter-gl/default.nix

```diff
@@ -18,9 +23,32 @@ buildPythonPackage rec {
     hash = "sha256-ObI8EEQ7mAOAuV6f+Ld4HH0xkFzqiAZqHDvzjwPA/wM";
   };
 
+  prePatch = ''
+    # Remove compiled TkGL, we compile it ourselves
+    rm -r src/tkinter_gl/tk
+    # Platform-specific directories are only necessary when using compiled TkGL
+    substituteInPlace src/tkinter_gl/__init__.py \
+      --replace-fail "pkg_dir = os.path.join(__path__[0], 'tk', sys.platform,)" \
+                     "pkg_dir = os.path.join(__path__[0], 'tk')"
+  '';
+
   build-system = [ setuptools-scm ];
 
-  dependencies = [ tkinter ];
+  buildInputs = [ libGL ];
+
+  dependencies = [
+    tkinter
+    pyopengl
```

Comment:
pyopengl isn't listed in upstream's dependencies. The reverse dependencies should depend on it:

> Clients need some other library such as PyOpenGL to make the OpenGL calls from python.